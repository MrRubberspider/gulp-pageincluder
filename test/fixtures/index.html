<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<title></title>
</head>
<body>
	<!-- include _header.html -->
	Lorem ipsum dolor sit amet, consectetur adipisicing elit. Illo, at saepe quasi quas facilis doloribus aspernatur nesciunt rem. Optio, autem, earum eos corporis laborum voluptas labore natus praesentium saepe facere.
</body>
</html>
