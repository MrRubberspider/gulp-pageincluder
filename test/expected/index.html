<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<title></title>
</head>
<body>
	This is the header.
	Lorem ipsum dolor sit amet, consectetur adipisicing elit. Illo, at saepe quasi quas facilis doloribus aspernatur nesciunt rem. Optio, autem, earum eos corporis laborum voluptas labore natus praesentium saepe facere.
</body>
</html>
